NEWS.md

```diff
@@ -7,6 +7,7 @@
 * Simplified `join_key` to better support primary keys.
 * `JoinKey` `R6` object was removed in favor of a list-like object with class name `join_keys`. Subset operators and assignments are supported (`[`, `[[`, `[<-` and `[[<-`)
 * `join_keys` function works as a constructor, getter and setter.
+* Extended `get_code.teal_data()` with a possibility to steer internal methods with `...` parameter. 
```

Comment:
It's actually under `teal.data 0.4.0`, please move up and add the `### Enhancements` section :slightly_smiling_face:

Reply:
Yes, totally - also mentioned this here https://github.com/insightsengineering/teal.data/pull/290#issuecomment-1933619133

Reply:
moved in here 8fec7927f8c2ad3856308474e751a16fe60d5e4e